dataset/components/schema_template/crud.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2022 Indoc Research
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+from typing import List
+from typing import Union
+from uuid import UUID
+
+from sqlalchemy import select
+from sqlalchemy.orm import contains_eager
+from sqlalchemy.sql import Select
+
+from dataset.components.crud import CRUD
+from dataset.components.dataset.models import Dataset
+from dataset.components.schema_template.models import SchemaTemplate
+
+
+class SchemaTemplateCRUD(CRUD):
+    """CRUD for managing schema template database models."""
+
+    model = SchemaTemplate
+
+    @property
+    def select_query(self) -> Select:
+        """Return base select including join with SchemaTemplate model."""
```

Comment:
Here `Dataset` is used in join:
```suggestion
        """Return base select including join with Dataset model."""
```